### PR TITLE
Feature/replace circle with ellipse

### PR DESCRIPTION
**Pull request by Bruswei**

### CODE REVIEW

File: h5p-bildetema-words-topic-image/src/utils/figure/figure.utils.test.ts [hunk 1]
```diff
@@ -6,28 +6,22 @@ describe(renderFigure.name, () => {
...
 
   it("should return an empty string if less than two points are provided", () => {
     const pointA: Point = { x: 0, y: 0 };
-
-    const actual1 = renderFigure([pointA]);
-    const actual2 = renderFigure([]);
-
+    const actual1 = renderFigure([pointA], 0, 0);
+    const actual2 = renderFigure([], 0, 0);
     expect(actual1).toBe("");
```
Comment (boyum) [inline h5p-bildetema-words-topic-image/src/utils/figure/figure.utils.test.ts:25]: ?

**Review by StianHaugland1 — APPROVED** [review]

File: h5p-bildetema-words-topic-image/src/utils/figure/figure.utils.test.ts (hunk 1 above)
Comment (Bruswei) [reply h5p-bildetema-words-topic-image/src/utils/figure/figure.utils.test.ts:25]: `Rotations` and `ellipseRadius` are the new parameters add for `renderFigure` which causing tests to fail.

Comment (Bruswei) [reply h5p-bildetema-words-topic-image/src/utils/figure/figure.utils.test.ts:25]: Should we have `Rotation` and `ellipseRadius` as optional instead?

Comment (Bruswei) [reply h5p-bildetema-words-topic-image/src/utils/figure/figure.utils.test.ts:25]: Keeping it as it for now. We can update tests later if needed.

**Pull request merged**